.github/workflows/pull-request-native.yml

```diff
@@ -36,12 +36,25 @@ jobs:
       uses: ./.github/actions/setup-runner
       with:
         more-memory: 'true'
-    - name: Setup Java, Maven, Gradle
+    - name: Setup Java, Gradle
       uses: ./.github/actions/dev-tool-java
 
-    - name: Build with Maven
-      run: ./mvnw --batch-mode --threads 1C install -DskipTests -am -pl :nessie-quarkus -pl :nessie-lambda
-
-    - name: Test native with Maven
-      run: |
-        ./mvnw --batch-mode install -Pnative,docker -Dtest.log.level=WARN -Dquarkus.native.native-image-xmx=6g -pl :nessie-quarkus -pl :nessie-lambda
+    - name: Gradle / integration test native
+      uses: gradle/gradle-build-action@v2
+      with:
+        # do not "push" the cache back, might "pollute" the cache for the main job
+        cache-read-only: true
+        arguments: |
+          :servers:quarkus-server:quarkusBuild
```

Comment:
why can't we run `./mvnw testNative` here? Also the previous version was testing `:nessie-lambda`